pkgs/applications/editors/vscode/extensions/contextmapper.context-mapper-vscode-extension/default.nix

```
@@ -0,0 +1,33 @@
+{ graphviz
+, jre
+, lib
+, makeWrapper
+, substituteAll
+, vscode-utils
+}:
+vscode-utils.buildVscodeMarketplaceExtension rec {
```

```suggestion
}:

vscode-utils.buildVscodeMarketplaceExtension rec {
```

Skip after arguments...

---

Sorry about that and thx for the help.